### ISSUE TITLE
Infinite loop after attempting to close devaultd's "repeat new password" inquiry

### ISSUE THREAD

**Issue by kyeno** [devaultcrypto/devault#121]
After cleaning testnet directory and wallet.dat, then starting devaultd clean, typing whatever in new password and then hitting ^C (ctrl+c) on "repeat password", devaultd runs into infinite printing loop saying that password must be at least 4 characters long. It then needs to be killed via SIGTERM
![Screen Shot 2019-05-07 at 11 07 05 AM](https://user-images.githubusercontent.com/1790957/57287979-e59bd800-70b8-11e9-8db4-471016d57d43.png)

**Comment by jonspock** [comment]
Not seeing this on Mac. Which OS?

**Comment by proteanx** [comment]
I'm also not able to reproduce on Linux. It just exits.
![Screenshot from 2019-05-07 11-13-06](https://user-images.githubusercontent.com/29043582/57311163-1ed44800-70b9-11e9-976a-9ca368256020.png)

**Comment by kyeno** [comment]
Happened to me on Linux32. Let me try again.

**Comment by kyeno** [comment]
Wrong bug report, sorry for that.
Ways to reproduce:
1. Open devaultd without testnet dir (or wallet.dat)
2. When asked, enter a 3 letter password
3. When you get a message the password should be 4 chars long, hit CTRL+C

**Comment by jonspock** [comment]
Please update and retry

**Comment by kyeno** [comment]
Apparently resolved via workaround.
